components/sync15/src/request.rs

```diff
@@ -500,6 +530,8 @@ mod test {
     use std::collections::VecDeque;
     use std::rc::Rc;
     use url::Url;
+
+    // test that the url is compiled correctly
```

Comment:
Can this comment be more specific? Or, alternatively, comment the test itself, since `test_url_building` is as meaningful to me as this comment.